tests/lib/ecma-features.js

```diff
@@ -34,22 +34,20 @@ const testFiles = shelljs
 // Tests
 //------------------------------------------------------------------------------
 
-describe('ecmaFeatures', () => {
+describe('ecma-features', () => {
   testFiles.forEach(filename => {
     // Uncomment and fill in filename to focus on a single file
-    // var filename = "jsx/invalid-matching-placeholder-in-closing-tag";
+    // const filename = "jsx/invalid-matching-placeholder-in-closing-tag";
```

Comment:
It would be better to just get rid of this line and the comment, it's left over from the mocha days - the more natural way to filter is using jest in interactive watch mode